packages/@expo/cli/src/utils/env.ts

```diff
@@ -254,3 +255,8 @@ class Env {
 }
 
 export const env = new Env();
+
+export function envIsStackblitz() {
```

Comment:
Would call this webcontainers instead of stackblitz and probably also set it as a default for an environment variable so it can be overwritten for testing purposes.